ui/src/components/radio-button/radio-button.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ChangeEventHandler, MouseEventHandler } from 'react';
+
+export interface RadioButtonProps {
+  /**
+   * The unique identifier for this component
+   */
+  id: string;
+  /**
+   * The name for the radioButton input field string
+   */
+  name: string;
+  /**
+   * Whether the component is rendered as a tile boolean
+   */
+  isTile?: boolean;
+  /**
+   * Event handler will be triggered when the radioButton value changes
+   * changeEventHandler<HTMLInputElement>
+   */
+  onChange: ChangeEventHandler<HTMLInputElement>;
+  /**
+   * Event handler will be triggered when the radioButton is clicked
+   * MouseEventHandler<HTMLInputElement>
+   */
+  onClick?: MouseEventHandler<HTMLInputElement>;
+  /**
+   * The text inside of the radioButton string
+   */
+  label: string;
+  /**
+   * The text below the label
+   */
+  subtext?: string;
+  /**
+   * Default value of the radioButton string | number | readonly string[]
+   */
+  value: string | number | readonly string[];
+  /**
+   * Whether the radioButton is checked by default boolean
+   */
+  checked: boolean;
+}
+const RadioButton: React.FC<RadioButtonProps> = ({
+  id,
+  name,
+  isTile,
+  onChange,
+  onClick,
+  label,
+  subtext,
+  value,
+  checked,
+}) => {
+  // Determine the class for styling based on `isTile` prop
+  const className = isTile ? 'radio-tile' : 'radio-button';
```

Comment:
I'm not seeing this `radio-tile` button anywhere in the CSS. I'd suggest removing the tile prop and references as we will be replacing this with MUI soon

Reply:
Scrapped this code from comet library, removed